include/CProducer.h

```diff
@@ -30,7 +31,7 @@ extern "C" {
 typedef struct CProducer CProducer;
 typedef int(*QueueSelectorCallback)(int size, CMessage *msg, void *arg);
```

Comment:
why need the parameter size, what's it function?

Reply:
QueueSelectorCallback is not asyn send  callback function